composer.json

```diff
@@ -6,9 +6,9 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.1",
         "drupal/config_filter": "^1.4",
-        "drupal/core": "^8.6"
+        "openeuropa/drupal-core-require-dev": "^8.6",
```

Comment:
Shouldn't this be `openeuropa/drupal-core-require`?

Reply:
Actually since this is going to be a contrib module we should not require "openeuropa/drupal-core-require". I'd also remove  "drupal/core": "^8.6" and leave our "openeuropa/drupal-core-require-dev" to decide that.

Reply:
We should also add in the module's info.yml file:

```
php: 7.1
```

As we need to account for sites to use this module without Composer.